packages/admin-panel/src/pages/resources/ResourcePage.js

```diff
@@ -62,6 +63,7 @@ export const ResourcePage = ({
         onProcessDataForSave={onProcessDataForSave}
         displayUsedBy={displayUsedBy}
       />
+      <LogsModal />
```

Comment:
This is a bit concerning to me... are we just going to end up with an ever growing list of modals defined here? That's why I want to consolidate into a single Modal component and then have its contents be customizable (since there can only be a single modal open at once). But it really was too much to bite off right now...

Reply:
Love the idea of consolidating them, but currently we have more than 5 modals, it is not an easy refactor.